src/Shared/E2ETesting/E2ETesting.targets

```diff
@@ -23,19 +19,46 @@
 
   <!-- Running prerequisites -->
 
-  <Target Name="EnsurePrerequisites" Condition="'$(EnforcePrerequisites)' == 'true'" BeforeTargets="EnsureNodeJSRestored">
+  <Target Name="EnsurePrerequisites">
```

Comment:
Much preferred the previous approach of only executing this target when `$(EnforcePrerequisites)` is enabled. As-is, the property only slightly changes the messages but doesn't make a real difference. There's a message already about `$(EnforcePrerequsites)` being disabled in `EnsureNodeJSRestored`. And, doubling all the messages makes the target harder to read.